packages/common/src/utils/filters.ts

```diff
@@ -77,9 +79,14 @@ export const filterableDimensionsOnly = (
     dimensions: Dimension[],
 ): FilterableDimension[] => dimensions.filter(isFilterableDimension);
 
-export const getFilterTypeFromField = (field: FilterableField): FilterType => {
-    const fieldType = field.type;
-    switch (field.type) {
+export const getFilterTypeFromItem = (item: FilterableItem): FilterType => {
+    if (isTableCalculation(item)) {
+        return FilterType.NUMBER;
```

Comment:
just to understand here what is happening, is the only expected type of table calc is number? or can it be string?

Reply:
yes, we currently assume the table calculation value is a number. There is a ticket to assign a type to table calculations https://github.com/lightdash/lightdash/issues/3469